packages/web/src/pages/upload-page/fields/ToggleRowField.tsx

```diff
@@ -1,4 +1,4 @@
-import { ComponentProps, PropsWithChildren } from 'react'
+import { ChangeEvent, ComponentProps, PropsWithChildren } from 'react'
```

Comment:
the name is a little strange here, usually i name something based off of the semantic html, which in this case seems like a switch? so maybe SwitchRowField?